src/xbps/packages/management.md

```diff
@@ -13,6 +13,10 @@ Most general package management is done with the following commands:
    configuration steps for installed packages, and can be used to reconfigure
    certain packages after changes in their configuration files. The latter
    usually requires the `--force` flag.
+- [xbps-alternatives(1)](https://man.voidlinux.org/xbps-alternatives.1) lists
+   and sets the alternatives provided by installed packages.
```

Comment:
Should we perhaps expand on what is meant by "alternatives" here?

Reply:
Probably. I will see if I can come up wth something. Thanks!